### ISSUE TITLE
Support for install scripts

### ISSUE THREAD

**Issue by wmantly** [robkinyon/ruby-packager#4]
Please include support for:

```
    --after-install FILE          A script to be run after package installation
    --before-install FILE         A script to be run before package installation
    --after-remove FILE           A script to be run after package removal
    --before-remove FILE          A script to be run before package removal
    --after-upgrade FILE          A script to be run after package upgrade. If not specified,
                                  --before-install, --after-install, --before-remove, and 
                                  --after-remove will behave in a backwards-compatible manner
                                  (they will not be upgrade-case aware).
                                  Currently only supports deb, rpm and pacman packages.
    --before-upgrade FILE         A script to be run before package upgrade. If not specified,
                                  --before-install, --after-install, --before-remove, and 
                                  --after-remove will behave in a backwards-compatible manner
                                  (they will not be upgrade-case aware).
                                  Currently only supports deb, rpm and pacman packages.
    --template-scripts            Allow scripts to be templated. This lets you use ERB to template your packaging scripts (for --after-install, etc). For example, you can do things like <%= name %> to get the package name. For more information, see the fpm wiki: https://github.com/jordansissel/fpm/wiki/Script-Templates
    --template-value KEY=VALUE    Make 'key' available in script templates, so <%= key %> given will be the provided value. Implies --template-scripts
```

**Comment by robkinyon** [comment]
Fixed in #6 and released as v0.1.2.

**Comment by robkinyon** [comment]
Note: I did *NOT* provide support for `--template-scripts` and `--template-value`. Please open another issue for those if that proves valuable.
